tutorials/admin/users/users.md

```diff
@@ -1 +1,399 @@
+your node. 
+Anyone who visits your node without logging in will have the same access as members of this group.
+
+This allows you to make part of your node *publicly available*, meaning a login is not required to access that
+resource.
+
+(users-group)=
+#### users
+
+This is a group that can be used to include every user with an account on your node. This group does not necessarily
+have any special permissions by default, but it is a useful way to update permissions for all users at once.
+
+In order to make use of this group, we recommend adding every new user to this group when they are added to the node.
+
+#### stac-admin
+
+Members of this group will have full read and write permissions to the [STAC catalog](../../users/catalog/catalog.md).
+Add users to this group who will need to update or manage the catalog.
+
+#### monitoring
+
+Members of this group will have full access to the Grafana and Prometheus [monitoring](../monitoring/monitoring.md) 
+components.
+
+This group is useful if you want to allow a user to be able to view sensitive monitoring data about your node without
+giving them full administrator permissions.
+
+(service-permissions)=
+## Service Permissions
+
+Marble [services](../services/services.md) are user facing tools that are deployed with the Marble software stack.
+Because they are user facing, you will likely want to configure which services are available to which users.
+
+As a first step you will want to decide which resources are accessible by users with accounts on your node (those in 
+the [users](users-group) group), and which will by publicly available (as defined by the [anonymous](anonymous-group) 
+group).
+
+Then you can add more groups and permission rules as you see fit. Some examples:
+
+- you want to host and serve some data that requires users to agree to terms and conditions before they get access.
+- you only want to provide access to the IDE to some users but not others.
+- you only want to allow some users to be able to create new 
+  [remote processing](../../users/remote-processing/remote-processing.md) workflows.
+
+### Managing Service Permissions
+
+To edit service permissions with Magpie, click the "Edit Services" button:
+
+```{image} images/magpie-edit-services-button.png
+:alt: Magpie Edit Services Button
+:width: 100px
+:align: center
+```
+
+This page shows each of the available service types:
+
+```{image} images/magpie-service-types.png
+:alt: Magpie Service Types
+:align: center
+```
+
+Here we will discuss the service types that are the most relevant for the Marble software. For a full description of all
+service types available through Magpie, see the 
+[Magpie documentation](https://pavics-magpie.readthedocs.io/en/latest/services.html#available-services).
+
+(service-permissions-api)=
+#### api
+
+This service type allows you to define access rules based on URL path segments.
+
+For example, the `STAC` service has the following endpoints:
+
+- stac/
+- stac/collections
```

Comment:
Does this pattern also include something like "stac/collections/{collection_id}/items"?

Reply:
Yes, I just used these as an example. I can throw an "etc." on the end to show that the list isn't exhaustive.